readme.md

```diff
@@ -19,6 +19,7 @@ Feel free to write to our [Support Center](http://devexpress.com/support/center)
 <a name="thetableofcontents"></a>
 ## Table of Contents
 
```

Comment:
Please make the following change below:

Line 34:
"Hilight a row on hover"  
Note that "Hilight" is mispelled. Please fix.